themes/bootstrap3/templates/combined/results.phtml

```diff
@@ -27,12 +25,7 @@
   $combinedResults = $this->results;
 
   // Set up breadcrumbs:
-  if (isset($this->overrideTitle)) {
-    $this->layout()->breadcrumbs = '<li class="active">' . $this->escapeHtml($this->overrideTitle) . '</li>';
-  } else {
-    $this->layout()->breadcrumbs = '<li class="active">' . $this->transEsc('Search') . ': ' .
```

Comment:
Is it significant that the breadcrumbs will now include the longer 'Search Results' pulled from the slot default above, rather than the abbreviated 'Search' found here?

Reply:
I don't think this is an issue, there is certainly room in the breadcrumbs.